### ISSUE TITLE
Mail address saved in admin settings only after pressing enter

### ISSUE THREAD

**Issue by LukasReschke** [owncloud/core#17094]
Confused the hell out of my users as everything was gone when he refreshed. Will prepare a fix.

**Comment by LukasReschke** [comment]
Mhm. This applies to all input fields in the admin settings. When clicking in a white area the input field is closed but the data not saved, though shown to the user.

@jancborchardt I guess we should adjust this everywhere?

**Comment by jancborchardt** [comment]
Duplicate of https://github.com/owncloud/core/issues/15176 (Confirm function for text fields) :)
